akka-cluster-tools/src/main/scala/akka/cluster/client/protobuf/ClusterClientMessageSerializer.scala

```diff
@@ -25,29 +25,33 @@ private[akka] class ClusterClientMessageSerializer(val system: ExtendedActorSyst
   private val GetContactsManifest = "B"
   private val HeartbeatManifest = "C"
   private val HeartbeatRspManifest = "D"
+  private val ReceptionistShutdownManifest = "E"
 
   private val emptyByteArray = Array.empty[Byte]
 
   private val fromBinaryMap = collection.immutable.HashMap[String, Array[Byte] ⇒ AnyRef](
     ContactsManifest → contactsFromBinary,
     GetContactsManifest → { _ ⇒ GetContacts },
     HeartbeatManifest → { _ ⇒ Heartbeat },
-    HeartbeatRspManifest → { _ ⇒ HeartbeatRsp })
+    HeartbeatRspManifest → { _ ⇒ HeartbeatRsp },
+    ReceptionistShutdownManifest → { _ ⇒ ReceptionistShutdown })
```

Comment:
For the record. Old clients will not recognize this new manifest when deserializing (server has been updated but client is old). That is fine, `fromBinary` will throw NotSerializableException which is treated as a transient error and the message will be discarded (with some warning logging).